src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetVersionsControllerTests.cs

```diff
@@ -302,6 +308,210 @@ public async Task ProcessorFailureStatusCode_Returns500()
         }
     }
 
+    public class UpdateVersionTests(
+        TestApplicationFactory testApp) : DataSetVersionsControllerTests(testApp)
+    {
+        [Theory]
+        [InlineData(DataSetVersionStatus.Failed)]
+        [InlineData(DataSetVersionStatus.Mapping)]
+        [InlineData(DataSetVersionStatus.Draft)]
+        [InlineData(DataSetVersionStatus.Cancelled)]
```

Comment:
Something like what I did in `CreatePreviewTokenControllerTests` might work well here?

```csharp
[MemberData(nameof(NonDraftDataSetVersionStatuses))]
```

Perhaps the definition  of the draft and non draft data set statuses could be put somewhere else, except that for Preview tokens I think draft really means only `DataSetVersionStatus.Draft`.

Reply:
Good idea

I've changed to using `MemberData` but have named it `UpdateableDataSetVersionStatuses` as I don't think `DraftDataSetVersionStatuses` would make sense anymore due to us only using `Draft` and `Mapping`. However, I'm also not completely happy with the name I have chose either, so I'm open to suggestions!

Reply:
Great!

I haven't really got any other suggestions for the name. I think you could drop the 'e' to make it `Updatable` and you could also drop `DataSet` to make it `UpdatableVersionStatuses`?

Or how about `EligibleUpdateDataSetVersionStatuses` or some variation of that? 🙂

Reply:
That was my poor spelling! 😆

I prefer your last suggestion! I've gone with `EligibleUpdateDataSetVersionStatuses` :)